src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,93 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int newCapacity = DEFAULT_INITIAL_CAPACITY;
+    private Node<K, V>[] table;
+    private int trashold;
```

Comment:
remove this field, in resize calculate it based on other fields. It's also misspelled